current_sensors/a2d_10_bit_converter.py

```diff
@@ -0,0 +1,16 @@
+from current_sensors.sensor import Sensor
+
+
+class A2D10Bit(Sensor):
```

Comment:
This is very similar to the 12 bit, and perhaps there is an assumption here that the 10 bit is bidirectional.
In reality, we may need to interface with 12 bit bidirectional as well, and also 14 bit sensors. That will result in the replication of very similar classes. Can you model it in terms of slope and intercept?